.github/workflows/ci_pullrequest.yml

```diff
@@ -0,0 +1,234 @@
+---
+# A lighter-weight version of CI specifically for pull requests
+name: "CI - Pull Request"
+concurrency:  # Cancel any existing runs of this workflow for this same PR
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+on:  # yamllint disable
+  pull_request:
+    paths-ignore:
```

Comment:
Similar question here - with a lighter-weight ci_pullrequest, do we still want there to be files/commits that don't trigger a rerun of ci?

Reply:
IMO let's pull it out and see how it goes. We can always add it back.

Reply:
Agreed